compiler/testData/debug/localVariables/localFun.kt

```diff
@@ -1,16 +1,54 @@
-//FILE: test.kt
 
```

Comment:
Change in file header looks strange

Reply:
same for newly added test

Reply:
Yeah, it looks a little off, but it was to create space for directives to appear at the top of the file.